src/MongoDB.EntityFrameworkCore/Serializers/BsonSerializerFactory.cs

```diff
@@ -282,7 +282,7 @@ private static IBsonSerializer GetCollectionSerializer(Type type, IBsonSerialize
         return CreateGenericSerializer(typeof(EnumerableInterfaceImplementerSerializer<,>), [type, itemType], childSerializer);
     }
 
-    internal static BsonSerializationInfo GetPropertySerializationInfo(IReadOnlyProperty property)
+    internal static BsonSerializationInfo GetPropertySerializationInfo(string? alias, IReadOnlyProperty property)
```

Comment:
We should make the alias optional and come after property rather than passing null in some paths.